pyvista/core/objects.py

```diff
@@ -469,10 +607,29 @@ def to_array(self):
         return np.flip(image.active_scalars.reshape(shape, order='F'), axis=1).swapaxes(1, 0)
 
     def plot(self, *args, **kwargs):
-        """Plot the texture as image data by itself."""
+        """Plot the texture as image data by itself.
+
+        If the texture is a cubemap, it will be displayed as a skybox.
+
+        Examples
+        --------
+        >>> from pyvista import examples
+        >>> texture = examples.download_masonry_texture()
+        >>> texture.plot()
+
+        """
+        if self.cube_map:
+            return self._plot_skybox(*args, **kwargs)
         kwargs.setdefault("rgba", True)
         return self.to_image().plot(*args, **kwargs)
 
+    def _plot_skybox(self, *args, **kwargs):
+        """Plot this texture as a skybox."""
+        pl = pyvista.Plotter(*args, **kwargs)
+        pl.add_actor(self.to_skybox())
+        pl.camera_position = kwargs.pop('cpos', 'xy')
```

Comment:
`cpos` is an invalid kwarg for `Plotter.__init__()` so we should pop this value off first:
```suggestion
        cpos = kwargs.pop('cpos', 'xy')
        pl = pyvista.Plotter(*args, **kwargs)
        pl.add_actor(self.to_skybox())
        pl.camera_position = cpos
```